src/stories/Spacing.mdx

```diff
@@ -0,0 +1,129 @@
+import { Meta, Story, Canvas, Controls } from '@storybook/blocks';
+import * as SpacingStories from './Spacing.stories.js';
+
+<Meta title="Foundation/Spacing" of={SpacingStories} />
+
+
+# Spacing
+
+
+[Figma Design Source](https://www.figma.com/file/tTu138pz3jLk0mykJSg3t4/KDS---Foundation-v1.0.0?node-id=4138%3A3703&mode=dev)
+
+
+## Example
+
+
+<style>
+{`
+.sbdocs-content {
+padding: 10px !important;
+}
+`}
+</style>
+
+
+<Canvas>
+<Story of={SpacingStories.Spacing} />
+</Canvas>
+
+
+### Controls
+
+
+<Controls of={SpacingStories.Spacing} />
+
+
+## Usage
+
+
+### SCSS Mixins
+
+
+```css
+@use '~@kyndryl-design-system/foundation/scss/mixins/spacing.scss';
+
+
+.your-css-selector {
+@include spacing.generate-spacings(padding, $kd-spacing-values);
+}
+
+
+.your-css-selector {
+@include spacing.spacing-top-bottom(padding, $kd-spacing-values);
```

Comment:
I don't think this works, since this mixin returns a CSS class. Any mixin used within a CSS class should just return the property. Need to make sure these are working as intended.

Reply:
Refactored and resolved